### PR TITLE
Added more classes.

### PR DESCRIPTION
**Pull request by muneebulrehman**
# Added classes and created associations
- Created `Classroom` class
- Created `Book` class
- Created `Rental` class
- Setup associations between the classes as required.

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  attr_reader :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(person, date)
+    Rental.new(slef, person, date)
```
Comment (t-yanick) [inline book.rb]: - [x] There is typo with `self`, please kindly correct that.

File: classroom.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_students(student)
+    @students.push(student)
```
Comment (t-yanick) [inline classroom.rb:11]: - [x] To implement the `has-many` of the relationship, when adding a student to a classroom it also sets the classroom for the student, for example: 
```ruby
  def add_student(student)
    @students << student
    student.classroom = self
  end
```
Please kindly fix that.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @muneebulrehman, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by muneebulrehman** [comment]
Thanks for pointing out.

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @muneebulrehman , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by muneebulrehman** [comment]
Thanks a lot @ShoiraTa

**Pull request merged**